garden/tutorials/sensors/sensor_launch.gz

```diff
@@ -1,11 +1,11 @@
 <?xml version='1.0'?>
-<ignition version='1.0'>
+<gz version='1.0'>
```

Comment:
This file should actually be called `sensor_launch.gzlaunch`. See example file names here https://github.com/gazebosim/gz-launch/tree/gz-launch6/examples

Reply:
perfect, was just a stab in the dark, never use the launches.

Reply:
Perfect, will do.